### PR TITLE
Feature/78 responsive update for the page of contact

### PR DESCRIPTION
**Pull request by Prom1s1ngYoung**
### Descriptions

- Link to UI design (if possible):

### Provide screenshots or videos that the function is working.

https://user-images.githubusercontent.com/110534329/226310360-73392d18-293f-4342-99d3-76dd8611696c.mov

https://user-images.githubusercontent.com/110534329/226310381-5c8b2947-b439-4d3a-a589-41a17b14ff30.mov

https://user-images.githubusercontent.com/110534329/226310395-e3b1cfbe-0751-40c8-b1fa-f01fd541071d.mov

### Provide screenshots of no warning, or error in the console.
The coverage issue is caused by other branch, fixed in another [card](https://github.com/taodemy/website/pull/133)
image

- [x] All unit tests passing and coverage of testing is more than 90% unless special needs.
- [x] No merge conflict with the `main` branch, make sure your code is up to date with `main`.
- [x] Check PR title is aligned with your ticket.
- [x] Check your app will run without crashes.
- [x] Check if there are hard-coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Remove unused comments.

### CODE REVIEW

**Review by tatoMa — COMMENTED** [review]
Is there any button link to this contact page?

File: constants/viewPortSize.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ enum EViewPortQuery {
   DESKTOP = "(min-width: 1024px)",
   TABLET = "(min-width: 743px) and (max-width: 1024px)",
   PHONE = "(max-width: 743px)",
+  OUROFFICES = "(max-width: 1118px)",
```
Comment (HarryuCATGREY) [inline constants/viewPortSize.ts:5]: Is that really necessary to use OUROFFICES? Others LGTM.

**Review by HarryuCATGREY — APPROVED** [review]

**Review by huchshm — APPROVED** [review]
LGTM

**Comment by Cameron933** [comment]
There is no link in the top navigation bar for contact page

**Comment by Prom1s1ngYoung** [comment]
[Figma](https://www.figma.com/file/q2ZHM8s0LcNxz3otGFLDzO/Website?node-id=1712-726&t=2kwQbGt2vDvnnElb-0)
Yes, there is no link in the top navigation bar. But there is a link to the contact page in the footer navigation bar.

**Pull request merged**